### PR TITLE
Feature document

### PR DESCRIPTION
**Pull request by vaaaaanquish**
# Update document for v1.0

TODO
- [x] Fix Warnings
- [x] ~Add Japanese documents~
- [x] Update English documents

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -84,7 +81,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
```
Comment (vaaaaanquish) [inline docs/conf.py:87]: WARNING because this directory does not exist.

File: docs/index.rst [hunk 2]
```diff
@@ -22,6 +22,7 @@ Table of Contents
    task_information
    task_settings
    slack_notification
+   using_task_cache_collision_lock
```
Comment (vaaaaanquish) [inline docs/index.rst:25]: WARNING because not index `using_task_cache_collision_lock.rst`

File: docs/slack_notification.rst [hunk 3]
```diff
@@ -12,22 +12,23 @@ Prepare following environmental variables:
    export SLACK_CHANNEL=channel-name     // not "#channel-name", just "channel-name"
 
 
-A Slack bot token can obtain from `here <https://api.slack.com/apps>`_.
+A Slack bot token can obtain from `slack app document <https://api.slack.com/apps>`_.
```
Comment (vaaaaanquish) [inline docs/slack_notification.rst:15]: WARNING because `here` is duplicated.

File: gokart/info.py [hunk 4]
```diff
@@ -3,7 +3,6 @@
 
 import luigi
 
-import gokart
```
Comment (vaaaaanquish) [inline gokart/info.py:6]: not used

File: gokart/task.py [hunk 5]
```diff
@@ -1,8 +1,6 @@
 import hashlib
 import os
-import sys
 import types
-from functools import partial
```
Comment (vaaaaanquish) [inline gokart/task.py:5]: not used

File: .readthedocs.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+# Build from the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# Explicitly set the version of Python and its requirements
+python:
+  version: 3.7
+  install:
+    - requirements: docs/requirements.txt
```
Comment (vaaaaanquish) [inline .readthedocs.yaml:11]: https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html#using-a-configuration-file

File: docs/conf.py [hunk 7]
```diff
@@ -1,4 +1,13 @@
 # -*- coding: utf-8 -*-
+
+# https://github.com/sphinx-doc/sphinx/issues/6211
+import luigi
+
+import gokart
+
+luigi.task.Task.requires.__doc__ = gokart.task.TaskOnKart.requires.__doc__
+luigi.task.Task.output.__doc__ = gokart.task.TaskOnKart.output.__doc__
```
Comment (vaaaaanquish) [inline docs/conf.py:9]: WARNING because undefined label for luigi label.

https://github.com/sphinx-doc/sphinx/issues/6211

File: pyproject.toml [hunk 8]
```diff
@@ -7,7 +7,7 @@ license = "MIT"
 readme = "README.md"
 homepage = "https://github.com/m3dev/gokart"
 repository = "https://github.com/m3dev/gokart"
-documentation = ""
+documentation = "https://gokart.readthedocs.io/en/latest/"
```
Comment (hirosassa) [inline pyproject.toml:10]: 👍

**Review by hirosassa — APPROVED** [review]
Thanks for your contribute! LGTM

**Pull request merged**